.github/workflows/tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: 'Set up Ruby 2.6'
         uses: actions/setup-ruby@v1
         with:
-          ruby-version: 2.6.3
+          ruby-version: 2.6
```

Comment:
this is '2.6.5' in our Gemfile

Reply:
Github seem to be recommending this. We could change it to `2.6.5` to make the requirement more strict - we would then get a failing build in the future if and when Github drop `2.6.5` like we did with `2.6.3`. What do you think?